params/params.go

```diff
@@ -0,0 +1,26 @@
+package params
+
+import (
+	"github.com/juju/bundlechanges"
+	"github.com/juju/httprequest"
+)
+
+// ChangesResponse contains the results of parsing a bundle into a list of
+// changes.
+type ChangesResponse struct {
+	Changes []bundlechanges.Change
```

Comment:
Is changing this to include the GUI args in the TODO list, not for this first branch?
We should not use the internal bundlechanges.Change struct here, but build an API specific response that also includes the values required by the GUI in order to build the uncommitted state for the bundle.
For an example, see how this is done in juju: https://github.com/juju/juju/blob/master/apiserver/client/bundles.go#L47